src/services/codefixes/fixAddMissingMember.ts

```diff
@@ -297,6 +297,6 @@ namespace ts.codefix {
             enumDeclaration.modifiers,
             enumDeclaration.name,
             concatenate(enumDeclaration.members, singleElementArray(enumMember))
-        ));
+        ), { useNonAdjustedStartPosition: false, useNonAdjustedEndPosition: true });

```

Comment:
Using `useNonAdjustedEndPosition: true` here will cause comments after the enum declaration to be duplicated.
```
const x = 2; // this is x

// this is E
enum E {
} // this was E
E.a;
```
will become
```
const x = 2; // this is x

// this is E
enum E {
    a
} // this was E
 // this was E
E.a;
```
However, using `useNonAdjustedEndPosition: false` will cause trailing whitespace and line breaks to disappear. Example:
```
const x = 2; // this is x

// this is E
enum E {
    a
}
E.a;
```
after the codefix becomes:
```const x = 2; // this is x

// this is E
enum E {
    a
}E.a;
```